### DISCUSSION TITLE
Support for adding a link in the error text

### DISCUSSION THREAD

**Discussion by bithaolee** [Shopify/function-examples#516] [Feedback: Cart and Checkout Validation API]
For the error message, sometimes it is necessary to guide the customer to the document to view, so if we can then add a clickable link in the error message, that would be good, many merchants have this demand.

It would be nice if the error alerts could support text in markdown format, for example:
```
Samples cannot be shipped to Australia, [view details](https://x.myshopify.com/page/delivery-notice)
```

**Comment by rchettupalli** [comment]
Yes, I am also looking for this feature. Is there any work around for this?
